SingularityUI/app/components/common/Navigation.jsx

```diff
@@ -35,6 +35,26 @@ function isActive(navbarPath, fragment) {
 // put into page wrapper, render children
 const Navigation = (props) => {
   const fragment = props.location.pathname.split('/')[1];
+  const navTitle = config.navTitleLinks
+    ? (
+      <ul className="nav navbar-nav">
```

Comment:
Wrapping this in the same `ul` structure will give the consistent menu look you seek.